lib/components/Atom/Calendar/date-picker.scss

```diff
@@ -0,0 +1,177 @@
+@import "@quill/quill.scss";
```

Comment:
Without this import, styles were not loaded on the first render, only after switching to another story/components. Not sure what I could do else, seems something wrong with settings/configuration

Reply:
cc: @prince-deriv